content/community/presentations/_index.nb.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Presentasjoner fra Altinn kaffe
-linktitle: Altinn kaffe
+title: Presentasjoner og Altinn kaffe
+linktitle: Presentasjoner
 description: The Altinn-kaffe meeting series is a low threshold meeting series, where app owners (and potential app owners) can learn more about what we're doing in Altinn 3.
```

Comment:
Should be translated.
(I left it untranslated in my PR, as I knew this PR was coming 😇 )